torch/export/_trace.py

```diff
@@ -1550,13 +1573,15 @@ def forward(self, *args, **kwargs):
                 gm, sig = aot_export(wrapped_mod, args, kwargs=kwargs, **flags)
                 log.debug("Exported program from AOTAutograd:\n%s", gm)
 
-            sig.parameters = pytree.tree_map(_strip_root, sig.parameters)
-            sig.buffers = pytree.tree_map(_strip_root, sig.buffers)
-            sig.inputs_to_buffers = pytree.tree_map(_strip_root, sig.inputs_to_buffers)
-            sig.inputs_to_parameters = pytree.tree_map(
-                _strip_root, sig.inputs_to_parameters
-            )
-            sig.buffers_to_mutate = pytree.tree_map(_strip_root, sig.buffers_to_mutate)
+            if sig:
+                sig.parameters = pytree.tree_map(_strip_root, sig.parameters)
```

Comment:
Hmm i feel like you need to this for export_for_training as well? We do this so that we can unwrap the top level module (because make_fx only accepts tuple output).

Reply:
I guess we don't produce a AOTAutograd graph signature in the training path yet. Gonna tag you in another comment.